libs-scala/contextualized-logging/BUILD.bazel

```diff
@@ -17,10 +17,12 @@ da_scala_library(
     scala_deps = [
         "@maven//:com_typesafe_akka_akka_actor",
         "@maven//:com_typesafe_akka_akka_stream",
-        "@maven//:org_scala_lang_modules_scala_collection_compat",
     ],
     scalacopts = ["-Xsource:2.13"],
     tags = ["maven_coordinates=com.daml:contextualized-logging:__VERSION__"],
+    versioned_scala_deps = {
+        "2.12": ["@maven//:org_scala_lang_modules_scala_collection_compat"],
+    },
```

Comment:
👍🏻

Reply:
good to see that removed/moved :+1: 

You should be able to remove the silencer plugin now again as it was only added because of the scala compat import.

Reply:
Just saw that comment. Thanks for the tip!